chain/network/src/routing/edge.rs

```diff
@@ -217,23 +218,34 @@ impl Edge {
     }
 }
 
-/// Edge object. Contains information relative to a new edge that is being added or removed
-/// from the network. This is the information that is required.
+/// Represents `edge` in Near Protocol P2P network.
```

Comment:
Tip: when writing a summary line, it's useful to avoid words already present in the name of the thing, to make the docs less tautological. 

I'd phrase it this way:

```
/// An `Edge` represents a direct connection between two peers in Near Protocol P2P network. 
///
/// Note that edge might either active or inactive. That is, we represent removed edge explicitly. (maybe some short note about *why* we need that)
```

Reply:
+1 - especially to the part why we're keeping 'removed' edges

Reply:
I added documentation about why we keep both `Active` and `Removed` edges.